Lib/test/test_ntpath.py

```diff
@@ -107,13 +108,13 @@ def test_splitdrive(self):
         tester('ntpath.splitdrive("//conky/mountpoint/foo/bar")',
                ('//conky/mountpoint', '/foo/bar'))
         tester('ntpath.splitdrive("\\\\\\conky\\mountpoint\\foo\\bar")',
-            ('', '\\\\\\conky\\mountpoint\\foo\\bar'))
+            ('\\\\\\conky', '\\mountpoint\\foo\\bar'))
```

Comment:
More than two slashes at the beginning of a path is always invalid. When the path is opened, the system collapses more than two slashes down to three slashes. The first two slashes get replaced with the NT device path `"\??\UNC\"`, so the path opens as `"\\??\\UNC\\\\"`, with a double backslash after the device name, which is always invalid. 

`splitdrive()` should check for more than two initial slashes on the root, and if so, just return the original path with an empty drive.

Reply:
I'd rather delegate that decision to the OS API, that way if it gets changed or improved in the OS we'll get the fix automatically. If we had a way to detect an invalid path here through an OS API without having to check whether it exists, we could use that, but I don't want to preempt the OS here anymore.